### ISSUE TITLE
Testing riot tags with `export default ...`

### ISSUE THREAD

**Issue by gburnett** [tompascall/riot-jest-transformer#5]
I changed the content of `hello.tag` in the demo to the following:
```
<hello>
  <p { ...attributes }></p>
  <script>
    export default {
      attributes: {
        id: 'my-id',
        role: 'contentinfo',
        class: 'main-paragraph'
      }
    }
  </script>
</hello>
```
This code was taken from an example on [riot.js.org](https://riot.js.org/documentation/#object-spread-attribute)

When I try to run `npm test` the tests fail to run and I get the following error:

```
SyntaxError: /Users/user/wrk/riot-jest-transformer/demo/hello.tag: 'import' and 'export' may only appear at the top level (3:3)

      1 | const riot = require("riot");
      2 | riot.tag2('hello', '<p 0></p>', '', '', function(opts) {
    > 3 |    export default {
        |    ^
      4 |      attributes: {
      5 |        id: 'my-id',
      6 |        role: 'contentinfo',

```

Can anyone advise me how to configure riot-jest-transform to work with this code?

Should I, for example, be adding arguments to `.riot-jest-transformer`?

**Comment by tompascall** [comment]
Try the default export out of the tag.

**Comment by gburnett** [comment]
I thought this was how riot components are defined.

[https://riot.js.org/api/#component-object](https://riot.js.org/api/#component-object)

I understand that the tests will run without that line of code.  I'm looking for advice on how to make them run with this line of code included.

**Comment by tompascall** [comment]
I'm on upgrading the transformer for riot v4 support, it may last about a few days.

**Comment by gburnett** [comment]
Great, I'll watch out for an update.

Thanks

**Comment by tompascall** [comment]
I've published a new version of the package (v4.0.0), pls check it out.

**Comment by gburnett** [comment]
Ok, I'm able to use this in the way I expected now.

Thank you very much.
